### PR TITLE
Allow to assign customer by app or staff 3.0

### PR DESCRIPTION
**Pull request by korycins**
I want to merge this change because it is a port of changes from: #7754

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Review by eCzerniak — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -138,7 +138,10 @@ All notable, unreleased changes to this project will be documented in this file.
 - Add `page_type_id` when it's possible for `AttributeValue` translations webhook. - #7825 by @fowczarek
 - Optimize available quantity loader. - #7802 by @fowczarek
 - Configure a periodic task for removing empty allocations - #7885 by @fowczarek
-
+- Add webhooks for stock changes: `PRODUCT_VARIANT_OUT_OF_STOCK` and `PRODUCT_VARIANT_BACK_IN_STOCK`  - #7590 by @mstrumeck
```
Comment (maarcingebala) [inline CHANGELOG.md:141]: Not part of this PR.

Comment (korycins) [reply CHANGELOG.md:141]: Sorry!
🚔
